franklin/github/views.py

```diff
@@ -136,42 +65,52 @@ def repository_list(request):
             github_repos = request.user.details.get_user_repos()
             # TODO - return error from github
             request.user.details.update_repos_for_user(github_repos)
-        sites = request.user.details.sites.all()
+        sites = request.user.details.sites.filter(is_active=True).all()
```

Comment:
`views.py` now only contains view logic. All "SDK" functions have been moved to `api.py`. Above you'll see that dashboard will only get sites that are active when it makes a request for all sites the user has currently deployed.